src/pages/Culture/Culture.tsx

```diff
@@ -1,42 +1,48 @@
+import { useEffect, useState } from 'react';
 import styled from 'styled-components';
 
 import AppDownIcon from '@/assets/svg/culture_app_box_download_ic_next.svg?react';
 import IconApp from '@/assets/svg/img_dive_app.svg?react';
 
+import { getCultureData } from '@/api/axios/Culture/cultureAxios';
+
 import CultureCard from './_components/CultureCard';
-import { CULTURE_DATA } from './_constants/cultureData';
+
+interface CultureInfo {
+  id: number;
+  description: string;
+  image: string;
+  name: string;
+  summary: string;
+  period: string;
+}
+
+interface Category {
+  name: string;
+  cultures: CultureInfo[];
+}
 
 function Culture() {
-  const { categories } = CULTURE_DATA; //객체를 가져오는 다른 방법!
+  const [categories, setCategories] = useState<Category[]>();
+
```

Comment:
예림님 useState에 인터페이스 활용 너무 잘하셨어요! 너무 굿~~~~ 지금 처럼 타입들을 interface로 분리하는 습관을 가지면 좋을거에요! 
그리고 useState를 사용할 때는 안정성을 위해 꼭 초기값을 설정해주는게 좋아요!
```suggestion
 const [categories, setCategories] = useState<Category[]>([]);
```

Reply:
오..!! 처음 안 사실이네요..!! 감사합니다!! 앞으로 useState 사용할때는 초기값을 꼭 넣어야겠어요..!! 반영완료했습니다!